build.gradle.kts

```diff
@@ -187,8 +185,6 @@ dependencies {
         exclude(group = "org.apache.logging.log4j", module = "log4j-core")
     }
 
-    integrationTestRuntimeOnly(libs.junit.platform.launcher)
-
```

Comment:
the above JUnit dependencies might conflict with the ones brought in by `hibernate-testing`. For that reason, we can simplify by keeping from specifying them in the first place.

Reply:
If we have a dependency, and Hibernate ORM happens to also have it, we should not avoid specifying that dependency in our `build.gradle.kts` because of a possible version conflict. If this conflict happens, we should resolve it in an appropriate way.

It is even more surprising that we specify the version of the `org.junit.platform:junit-platform-launcher` dependency, but do not specify this dependency.

Reply:
I reverted back the changes related to JUnit. I found bumping JUnit dependencies to latest version would break our integration testing due to version conflicts. It takes time to fix it but I don't think it is worth while currently.